src/ui/window_management.rs

```diff
@@ -54,7 +54,7 @@ pub fn opening_window<V: IntoView + 'static>(
 	view: impl Fn() -> V + 'static,
 	spec: WindowSpec,
 	size: Size,
-	movable_by_window_background: bool,
+	_movable_by_window_background: bool, // movable_by_window_background is too buggy to enable just yet
```

Comment:
yeah I commented it out here. It's too buggy including for sliders etc. Issue for it here: https://github.com/lapce/floem/issues/490